hera_pspec/pstokes.py

```diff
@@ -103,23 +106,114 @@ def _combine_pol(uvd1, uvd2, pol1, pol2, pstokes='pI', x_orientation=None):
     assert isinstance(uvd2, pyuvdata.UVData), \
         "uvd2 must be a pyuvdata.UVData instance"
 
+    for i, uvd in enumerate([uvd1, uvd2]):
+        try:
+            uvd.pol_convention
+        except AttributeError:
+            warnings.warn(
+                f'No polarization convention in uvd{i+1}. '
+                'Considering it to be "avg".')
+            setattr(uvd, 'pol_convention', 'avg')
```

Comment:
This is not quite correct -- the attribute can be defined, but set to `None`. So, I would do 

```python
pol_convention = getattr(uvd, 'pol_convention', None)
if pol_convention is None:
    warnings.warn(...)
```